### ISSUE TITLE
CSS Grid doesn't work with "serde" feature

### ISSUE THREAD

**Issue by jihiggins** [DioxusLabs/taffy#362]
## `taffy` version

0.3.0

## Platform

Rust

## What you did

Enable the serde feature and try to build

## What went wrong

```
91   |   #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
     |                                        ^^^^^^^^^ the trait `Serialize` is not implemented for `TrackSizingFunction`
```
etc

## Additional information

If it's fine to just add cfg_attr derives to the grid structs/enums, I can throw up a PR for it. Mostly just want to make sure that I'm not missing something / that it would be the correct fix.

**Comment by nicoburns** [comment]
Ah, I was someone under the impression that the `serde` had been removed inmhttps://github.com/DioxusLabs/taffy/pull/84, but it turns out that was just the `serde_camel_case` and `serde_kebab_case` features which were separate.

> If it's fine to just add cfg_attr derives to the grid structs/enums, I can throw up a PR for it. Mostly just want to make sure that I'm not missing something / that it would be the correct fix.

Yep, that seems like the correct fix to me (we probably also ought to add a CI run checking the serde feature to prevent regressions). PR would be welcome!

**Comment by nicoburns** [comment]
@jihiggins This has now been released in Taffy `0.3.1`. Let us know if you have any further issues!
